File: C Programs/Assignment 4/reverse_array.c
/*2. Accept an array of 5 elements and 
create its reverse in same array and then print the array.*/

#include<stdio.h>
int main()
{
	int arr[5],brr[5];
	read_array(arr);
	printf("\nEntered array is \n");
	display(arr);
	reverse(arr,brr);
	printf("\n\nAfter reverse array is ");
	display(brr);
	return 0;
	
	
}
void read_array(int arr[])
{
	int i;
	printf("Enter elements of the array \n");
	for(i=0;i<=4;i++)
	{
		scanf("%d",&arr[i]);
	}	
}

void display(int arr[])
{
	int i;
	for(i=0;i<=4;i++)
	{
	printf("%d\t",arr[i]);
    }
}
void reverse(int arr[],int brr[])
{
	int i,e=4;
	for(i=0;i<=4;i++)
	{
		brr[e]=arr[i];
		e--;
	}
}

File: Basic Java/Basic/ReverseArray.java
package com.cdac.iet;/*. Write a Java Program to Create an array and write a function to reverse the array*/



import java.util.Scanner;


public class ReverseArray
{
	public static void main(String args[])
{
	int []arr =new int[5];
	System.out.println("Enter 5 elements of array");
	Scanner sc=new Scanner(System.in);
	for(int i=0;i<arr.length;i++)
	{
	 arr[i]=sc.nextInt();
	}

 	display(arr);
	reverse(arr);
}
static void display(int arr[])
{
	System.out.println("Array is:");	
	for(int i=0;i<arr.length;i++)
	{
	System.out.printf("\t%d",arr[i]);
	}
}

static void reverse(int arr[])
{
	int brr[]=new int[5];
	int j=4;
	for(int i=0;i<5;i++)
	{
		brr[j]=arr[i];
		j--;
	}
	System.out.println("\nReverse Array is:");	
	display(brr);
}


}